src/pytorch_lightning/strategies/ipu.py

```diff
@@ -45,6 +46,7 @@ class LightningIPUModule(_LightningModuleWrapperBase):
     def __init__(
         self, pl_module: Union["pl.LightningModule", _LightningPrecisionModuleWrapperBase], precision: Union[str, int]
     ) -> None:
+        rank_zero_deprecation("`LightningIPUModule` is deprecated in v1.8 and will be removed in v2.0.0")
```

Comment:
```suggestion
        rank_zero_deprecation("`LightningIPUModule` is deprecated in v1.8 and will be removed in v1.10")
```

Reply:
If we merge it as bugfix before 1.7 is release, we should decrement the numbers here. Waiting for reviews and decision.

Reply:
I propose we just give this a 1 version deprecation.

And I agree this can be considered a bug-fix. So let's assume it will be deprecated in `v1.7.x` and removed in `v1.8`.

Meaning we update these messages, cherry-pick into a bug-fix release after merging this PR, and then open a new PR removing the code from master.

Reply:
I've updated this to deprecate in 1.7 and remove in 1.8